### ISSUE TITLE
Globally defined simd.f32x16 causes compiler crash

### ISSUE THREAD

**Issue by Skytrias** [odin-lang/Odin#4267]
## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System & Odin Version:
        Odin:    dev-2024-09:8814170ed
        OS:      Windows 10 Home Basic (version: 22H2), build 19045.4780
        CPU:     AMD Ryzen 5 3600 6-Core Processor
        RAM:     16310 MiB
        Backend: LLVM 18.1.8

## Expected Behavior

Compile as expected - Globals should behave like locals

## Current Behavior

Causes the compiler to crash

### Steps to Reproduce

```odin
package main

import "core:simd"

WIDE_ZERO: simd.f32x16 = f32(0)
WIDE_ONE: simd.f32x16 = f32(1)
WIDE_TWO: simd.f32x16 = f32(2)
WIDE_THREE: simd.f32x16 = f32(3) 

main :: proc() {}
```

When they are defined as `Constants` it doesnt crash.

### Failure Logs

```
LLVM Error:
Global variable initializer type does not match global variable type!
ptr @graphics.WIDE_ZERO
Global variable initializer type does not match global variable type!
ptr @graphics.WIDE_ONE
Global variable initializer type does not match global variable type!
ptr @graphics.WIDE_TWO
Global variable initializer type does not match global variable type!
ptr @graphics.WIDE_THREE
```

**Comment by Feoramund** [comment]
The sample given as such does not cause the compiler error reported, unless you use the variables, such as in a print statement.

**Comment by Skytrias** [comment]
> The sample given as such does not cause the compiler error reported, unless you use the variables, such as in a print statement.

yep i think the workaround was to not make them const, then it worked. not sure if locally defined consts worked or not, been a while.

**Comment by Kelimion** [comment]
I still get this error if I try to print a global variable:
```odin
LLVM Error:
Global variable initializer type does not match global variable type!
ptr @"main::WIDE_ONE"
```

**Comment by corleypc** [comment]
No crash and prints fine. Likely fixed.
